### ISSUE TITLE
Make paths configurable

### ISSUE THREAD

**Issue by jskacel** [lucas-benedito/sos-ansible#18]
Would be nice to be able to set default path for: sosreports, rules, output files

**Comment by lucas-benedito** [comment]
At the moment, the following is the default for sosreports location and rules file
```
SOS_DIRECTORY = os.path.abspath("/tmp/test_sosreport/")
RULES_FILE = os.path.abspath("/tmp/rules/rules.json")
```

**Comment by jskacel** [comment]
Changed title and also first comment to be more generic

**Comment by lucas-benedito** [comment]
Will look into creating a config file. Tried to keep simple to start as this creates more points of failure ( incorrect syntax, missing config file etc)

**Comment by jskacel** [comment]
I need to learn python finally, but logic more less:
if config_file exists:
 load-it
else
  default_values
fi

if it's different then default, I would print them in every run like ansible has that:
config_file: blabla
path to this: blabla

**Comment by lucas-benedito** [comment]
Addressed with #23 but requires further development. Expanding the use input with "~" is something we are looking at.  A new issue will be opened for that behavor
